tests/foreman/ui/test_settings.py

```diff
@@ -489,3 +489,41 @@ def test_negative_update_hostname_with_empty_fact(session):
         finally:
             for key, value in default_hostname.items():
                 setting_cleanup(setting_name=key, setting_value=value.value)
+
+
+@tier3
+def test_positive_entries_per_page(session):
```

Comment:
Since you modify global setting, I'd advise to use `run_in_one_thread` decorator.

Reply:
Fixed.